packages/client-core/src/admin/adminRoutes.tsx

```diff
@@ -6,8 +6,8 @@ Version 1.0. (the "License"); you may not use this file except in compliance
 with the License. You may obtain a copy of the License at
 https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
 The License is based on the Mozilla Public License Version 1.1, but Sections 14
-and 15 have been added to cover use of software over a computer network and 
-provide for limited attribution for the Original Developer. In addition, 
+and 15 have been added to cover use of software over a computer network and
+provide for limited attribution for the Original Developer. In addition,
```

Comment:
Hey Andres, can you recheck your vscode editorconfig or prettier config for which these lines are getting changed during commits?
(with them absent, your PR looks more concise and shorter)

Reply:
done! thx